packages/file_selector/file_selector_macos/macos/Classes/FileSelectorPlugin.swift

```diff
@@ -96,16 +99,31 @@ public class FileSelectorPlugin: NSObject, FlutterPlugin, FileSelectorApi {
     }
 
     if let acceptedTypes = options.allowedFileTypes {
-      var allowedTypes: [String] = []
-      // The array values are non-null by convention even though Pigeon can't currently express
-      // that via the types; see messages.dart.
-      allowedTypes.append(contentsOf: acceptedTypes.extensions.map({ $0! }))
-      allowedTypes.append(contentsOf: acceptedTypes.utis.map({ $0! }))
-      // TODO: Add support for mimeTypes in macOS 11+. See
-      // https://github.com/flutter/flutter/issues/117843
-
-      if !allowedTypes.isEmpty {
-        panel.allowedFileTypes = allowedTypes
+      if #available(macOS 11, *), !forceLegacyTypes {
+        var allowedTypes: [UTType] = []
+        // The array values are non-null by convention even though Pigeon can't currently express
+        // that via the types; see messages.dart.
```

Comment:
Here and below: is there a pigeon bug for that? If so, perhaps link here to make searching for these easier later on when we fix it.

Reply:
Done.

Reply:
Done